tests/emails/offerer_validation_test.py

```diff
@@ -0,0 +1,200 @@
+    assert email['Subject'] == 'Validation de votre structure et de compte administrateur rattaché'
+
+
+@clean_database
+def test_make_validation_confirmation_email_offerer_user_offerer_editor(app):
+    # Given
+    user = create_user(email='admin@letheatresas.com')
+    offerer = create_offerer(name='Le Théâtre SAS')
+    user_offerer = create_user_offerer(user, offerer)
+    # When
+    with patch('utils.mailing.find_user_offerer_email', return_value='editor@letheatresas.com'):
+        email = make_validation_confirmation_email(user_offerer, offerer)
+    # Then
+    email_html = BeautifulSoup(email['Html-part'], 'html.parser')
+    html_validation_details = str(email_html.find('p', {'id': 'validation-details'}))
+    assert 'Votre structure "Le Théâtre SAS"' in html_validation_details
+    assert 'L\'utilisateur editor@letheatresas.com' in html_validation_details
+    assert 'en tant qu\'éditeur' in html_validation_details
+    assert email['FromName'] == 'pass Culture pro'
+    assert email['Subject'] == 'Validation de votre structure et de compte éditeur rattaché'
+
+
+@clean_database
+def test_make_validation_confirmation_email_user_offerer_editor(app):
+    # Given
+    user = create_user(email='admin@letheatresas.com')
+    offerer = create_offerer(name='Le Théâtre SAS')
+    user_offerer = create_user_offerer(user, offerer)
+    # When
+    with patch('utils.mailing.find_user_offerer_email', return_value='editor@letheatresas.com'):
+        email = make_validation_confirmation_email(user_offerer, offerer=None)
+    # Then
+    email_html = BeautifulSoup(email['Html-part'], 'html.parser')
+    html_validation_details = str(email_html.find('p', {'id': 'validation-details'}))
+    assert 'Votre structure "Le Théâtre SAS"' not in html_validation_details
+    assert 'L\'utilisateur editor@letheatresas.com a été validé' in html_validation_details
+    assert 'en tant qu\'éditeur' in html_validation_details
+    assert email['FromName'] == 'pass Culture pro'
+    assert email['Subject'] == 'Validation de compte éditeur rattaché à votre structure'
+
+
+@clean_database
+def test_make_validation_confirmation_email_offerer(app):
+    # Given
+    offerer = create_offerer(name='Le Théâtre SAS')
+    # When
+    with patch('utils.mailing.find_user_offerer_email', return_value='admin@letheatresas.com'):
+        email = make_validation_confirmation_email(user_offerer=None, offerer=offerer)
+    # Then
+    email_html = BeautifulSoup(email['Html-part'], 'html.parser')
+    html_validation_details = str(email_html.find('p', {'id': 'validation-details'}))
+    assert 'Votre structure "Le Théâtre SAS"' in html_validation_details
+    assert 'L\'utilisateur' not in html_validation_details
+    assert email['FromName'] == 'pass Culture pro'
+    assert email['Subject'] == 'Validation de votre structure'
+
+
+@mocked_mail
+@clean_database
+def test_write_object_validation_email_should_have_some_specific_information(app):
```

Comment:
`should_have_some_specific_information` n'apporte aucune information

```suggestion
def test_validation_email_object(app):
```